src/Parsing/Impl/Extensions/AstExtensions.cs

```diff
@@ -20,16 +20,16 @@
 
 namespace Microsoft.Python.Parsing.Extensions {
     public static class AstExtensions {
-        public static IEnumerable<Node> ChildNodesDepthFirst(this Node node)
+        public static IEnumerable<INode> ChildNodesDepthFirst(this INode node)
```

Comment:
I'm not sure that this is backwards compatible, since this changes the signature of non-internal stuff. Do we need to retain the `Node` variants? @MikhailArkhipov

Reply:
@heejaechang says this isn't backwards compatible. You'll have to bring back the old definitions, it seems.

Reply:
it is changing signature which is explicitly stated in IL, runtime will throw method not found exception.

Reply:
So do you think I should just duplicate the methods but have them for node as well?

Reply:
Yes, you can just leave the old code and also have the new code. Copy/paste. I tested this and the compiler seemed fine with it.

Reply:
Alright sounds good!